.gitignore

```diff
@@ -1,2 +1,3 @@
 /composer.lock
 /vendor
+/.phpunit.result.cache
```

Comment:
Missing line ending here. Also for info, I prefer to add this to my global git ignore. But I've no problem adding it here too, just letting you know for other projects

Reply:
Removed this as you recommended